libraries/core-react/src/components/Group/Group.tsx

```diff
@@ -0,0 +1,36 @@
+import { forwardRef } from 'react'
+import styled from 'styled-components'
+import { group as tokens } from './Group.tokens'
+
+const { border } = tokens
+
+const GroupBase = styled.div`
```

Comment:
Suggest changing this to use the function style (`styled.div(() => css``)`), then you only need to do the `border.type === 'border' && border.radius` check once as a const.